phpcs.xml

```diff
@@ -2,23 +2,41 @@
 <ruleset name="WirecardOxidRuleset">
   <description>Wirecard OXID extension</description>
 
-  <!-- Ignore patterns -->
-  <exclude-pattern>/.bin/</exclude-pattern>
-  <exclude-pattern>/phpmd/</exclude-pattern>
-  <exclude-pattern>*/de/</exclude-pattern>
-  <exclude-pattern>*/en/</exclude-pattern>
-  <exclude-pattern>Tests/*</exclude-pattern>
+  <!-- Load external standards -->
+  <config name="installed_paths" value="phpcs,vendor/oxid-esales/coding-standards,vendor/escapestudios/symfony2-coding-standard"/>
 
-  <rule ref="vendor/oxid-esales/coding-standards/Oxid/ruleset.xml">
-  </rule>
+  <!-- Exclude patterns -->
+  <exclude-pattern>.bin</exclude-pattern>
+  <exclude-pattern>.bundle</exclude-pattern>
+  <exclude-pattern>ops</exclude-pattern>
+  <exclude-pattern>out</exclude-pattern>
+  <exclude-pattern>phpcs</exclude-pattern>
+  <exclude-pattern>phpmd</exclude-pattern>
+  <exclude-pattern>Tests</exclude-pattern>
+  <exclude-pattern>translations</exclude-pattern>
+  <exclude-pattern>vendor</exclude-pattern>
+  <exclude-pattern>views</exclude-pattern>
 
-  <rule ref="WirecardOxidRuleset/Sniffs/Functions/FunctionNameUnderscoreSniff.php">
-  </rule>
+  <!-- Inherit OXID's standard -->
+  <rule ref="Oxid"/>
+
+  <!-- Add custom standard -->
+  <rule ref="Custom"/>
 
+  <!-- Add sniffs from other standards -->
+  <rule ref="Generic.Arrays.DisallowLongArraySyntax"/>
+  <rule ref="Generic.Formatting.SpaceAfterCast"/>
   <rule ref="Generic.Files.LineLength">
     <properties>
       <property name="lineLimit" value="120"/>
       <property name="absoluteLineLimit" value="120"/>
     </properties>
   </rule>
+  <rule ref="Generic.Metrics.CyclomaticComplexity">
+    <properties>
+      <property name="complexity" value="5"/>
+      <property name="absoluteComplexity" value="5"/>
```

Comment:
Maybe set to `4`?

Reply:
The measurement for PHPCS is one higher than that of PHPMD. So this is equal to a complexity of `4` for PHPMD.